src/expression-engine/models/content/resolver.js

```diff
@@ -103,6 +103,21 @@ export default {
     body: ({ body, legacy_body }, _, { models }) => models.Content.cleanMarkup(body || legacy_body),
     description: ({ description }) => description,
     ooyalaId: ({ video }) => video,
+    video: ({ video }) => {
+      // Todo: should these be stored as ENV vars?
+      const pbid = 'ZmJmNTVlNDk1NjcwYTVkMzAzODkyMjg0&pcode=';
+      const pcode = 'E1dWM6UGncxhent7MRATc3hmkzUD';
+
+      // Todo: this is hosted on a personal dropbox. Should be moved to S3 or something (needs CORs though)
+      const playerConfig = 'https%3A%2F%2Fdl.dropbox.com%2Fs%2Fsodcv1d9a4ezwm4%2Fskin.new.json%3Fdl%3D1';
```

Comment:
you can use this instead
`//d3n6tjerleuu41.cloudfront.net/newspring/skin.new.json`	



config: "//d3n6tjerleuu41.cloudfront.net/newspring/skin.new.json"
52